src/libertem/udf/blobfinder/correlation.py

```diff
@@ -495,12 +519,34 @@ def postprocess(self):
 
 
 def run_fastcorrelation(ctx, dataset, peaks, match_pattern: MatchPattern, roi=None):
+    """
+    Wrapper function to construct and run a :class:`FastCorrelationUDF`
+
+    Returns
+    -------
+    buffers : Dict[libertem.common.buffers.BufferWrapper]
+        See :meth:`CorrelationUDF.get_result_buffers` for details.
+    """
     peaks = peaks.astype(np.int)
     udf = FastCorrelationUDF(peaks=peaks, match_pattern=match_pattern)
     return ctx.run_udf(dataset=dataset, udf=udf, roi=roi)
 
 
 def run_blobfinder(ctx, dataset, match_pattern: MatchPattern, num_peaks, roi=None):
+    """
+    Wrapper function to find peaks in a dataset and refine their position using
+    :class:`FastCorrelationUDF`
+
+    Returns
+    -------
+    sum_result : numpy.ndarray
+        Log-scaled sum frame of the dataset
+    centers, refineds, peak_values, peak_elevations : libertem.common.buffers.BufferWrapper
+        See :meth:`CorrelationUDF.get_result_buffers` for details.
+    peaks : numpy.ndarray
+        List of found peaks
+    """
+    # FIXME use ROI for sum!
```

Comment:
Noticed this FIXME: you can now pass the `roi` to `Context.run` as the sum is implemented as a UDF! Also, if possible, the parameters should be documented